kartothek/core/dataset.py

```diff
@@ -855,6 +851,7 @@ def create_partition_key(
 
         returns 'my-uuid/testtable/index1=value1/index2=value2/data'
     """
+    # TODO: ensure private
```

Comment:
Still to be done in this PR?

Reply:
I'll remove the TODO. What's public and what isn't was effectively taken care of in #435

Reply:
- [ ] remove todo